src/components/SearchResults/SearchResults.scss

```diff
@@ -0,0 +1,84 @@
+$highlight-background: #f2f8fb;
+
+.search-result-card {
```

Comment:
Can probably combine this with .card-link once you make the <Link> tag the outermost element.